### ISSUE TITLE
Turnkey tracker wiki is very outdated

### ISSUE THREAD

**Issue by OnGle** [turnkeylinux/tracker#975]
What it says in the title.

**Comment by JedMeister** [comment]
Yes good point @OnGle - it's not a priority, but it would be good to clean it up a little. I don't really like deleteing content provided by users, but OTOH, if it's so outdated that ti's no longer relevant or useful, then we probably should delete, or at least edit it so it's really clear it's out of date.
